dockerfiles/cli/scripts/cmd_init.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (c) 2016 Codenvy, S.A.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+
+
+cmd_init_reinit_pre_action() {
```

Comment:
It's not a typo - I checked it a couple times.  It's a special hook that is only called if you are *not* reiniting the system.